packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -50,9 +52,9 @@ import {
  */
 export interface ContainerDevtoolsProps extends HasContainerKey {
 	/**
-	 * The Container to register with the Devtools.
+	 * The decomposed container to register with the Devtools.
```

Comment:
Let's revert this. Conceptually, this is a "container". The implementation details around the type in use shouldn't matter to the end user.